lib/fetch.js

```diff
@@ -55,6 +54,32 @@ export function mapAndSanitate(item, type) {
   };
 }
 
+export function sanitize(key, type, defaultValue, json) {
```

Comment:
`sanitize` doesn't seem like the right name. This function gets a value from an object, nothing is sanitized. `get` ?

arg 4 isn't JSON, could we rename the var?

What's the convention for arg order? In non-curried languages I like `(haystack, needle, options)`